src/Toml/Printer.hs

```diff
@@ -167,3 +172,26 @@ prettyTables options i pref =
 
 prettyKey :: Key -> Text
 prettyKey (Key k) = Text.intercalate "." $ map unPiece (NonEmpty.toList k)
+
+prettyTableArrays :: PrintOptions -> Int -> Text -> HashMap Key (NonEmpty TOML) -> Text
+prettyTableArrays options i pref =
+    Text.intercalate "\n" . map arrText . orderWith options . HashMap.toList
```

Comment:
The following piece of code appears 3 times already:

```haskell
map somethingSomething . orderWith options . HashMap.toList
```

Looks like it can be moved into separate function.